src/middlewares/auth/authStrategies.js

```diff
@@ -1,12 +1,30 @@
-const { User, UserStatuses, UserRoles } = require('../../models');
+const { User, UserStatuses } = require('../../models');
+const { UsersService } = require('../../services');
 const bcrypt = require('bcryptjs');
+
 require('dotenv').config();
 
-const verifyCustom = async (req, done) => {
-  const user = await User.findOne({ email: req.body.email }).exec();
+const usersService = new UsersService();
+
+const checkUser = async (req) => {
+  const user = await User.findOne({ email: req.email }).exec();
   const status = await UserStatuses.findOne({ name: 'Active' }).exec();
```

Comment:
Lets move this call to userStatuses service and call it from here. Let's follow pattern: each service responsible for his model, so we can make calls to model only through service

Reply:
I agree with you. The problem is that there's no separate function in user services to find user status. It's a big mess there.